challenges/objects-arrays.js

```diff
@@ -106,14 +150,24 @@ The zoos are concenred about animals with a lower population count. Find out whi
 
 */
 const largerPopulation = [];
+zooAnimals.filter((animal) => {
+  if (animal.population < 5) {
+    largerPopulation.push(`${animal.animal_name} has a population lower than 5!`);
```

Comment:
When using .filter() remember not to use .push() with it as well as .filter already returns that population lower than 5 in one line. Good job getting this to work though.